test cases/java/9 jdk/src/meson.build

```diff
@@ -0,0 +1,17 @@
+jdkjar = jar(
+  'jdkjar',
+  'com' / 'mesonbuild' / 'JdkTest.java',
+  main_class : 'com.mesonbuild.JdkTest',
+)
+
+test(
+  'jdktest',
+  java,
+  args: [
+    '-Djava.library.path=@0@'.format(fs.parent(jdkjava.full_path())),
+    '-jar',
+    jdkjar,
```

Comment:
Does this jdkjar not create an implict dependency on the target and that is why the test is failing?

Reply:
It looks like your building a 32bit .dll and running witha  64bit java

Reply:
Can I force a 64 bit build of the .dll? I don't do Windows development.

Reply:
That image is a 32bit image, so we should be able to get a 32bit java or we need to skip running the test as it's not going to work.